Things.md

```diff
@@ -169,4 +170,13 @@ Events can also be added to repeat, similar to using `setInterval`.  A new third
 FSM.addEventInterval(FSM.shiftHoriz, 100, 5, thing, 2);
 ```
 
-More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
\ No newline at end of file
+More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
+
+# Thing classes
+
+Classes are generated using [ObjectMakr](https://github.com/FullScreenShenanigans/ObjectMakr), a module that dynamically generates the inheritance structure of classes. It takes in the structure and a list of properties and the class objects are then constructed using this information. 
+
+When a FullScreenShenanigans game starts up, ObjectMakr receives the inheritance structure and properties list from an `objects.js` file. It then generates prototypes for all of the internal classes (Box, Tree, etc.). Most of the basic properties, like `height`, `width`, and `opacity` are defined in the parent Thing class. ObjectMakr allows you to easily initialize a new instance of a class and assign properties to it in one step. 
```

Comment:
> ObjectMakr allows you to easily...

Sounds a little infomercial-y. Say "ObjectMakr's addThing is a shortcut to..." instead of that.